src/main/java/de/hpi/bpt/argos/util/RestEndpointUtilImpl.java

```diff
@@ -153,7 +153,8 @@ public void logSendingResponse(Logger logger, Request request, int responseStatu
 			message = responseMessage;
 		}
 
-		logger.info(String.format("sending response: '%1$s' -> %2$d -> %3$d bytes of response",
+		logger.info(String.format("%1$s\t<-\t'%2$s' -> %3$d -> %4$d bytes of response",
+				request.ip().toString(),
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') "ip" returns a string, there's no need to call "toString()". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AS1858)